data/plugins/treeview.lua

```diff
@@ -862,12 +875,11 @@ command.add(
       text = text,
       submit = function(filename)
         local doc_filename = item.dir_name .. PATHSEP .. filename
-        core.log(doc_filename)
-        local file = io.open(doc_filename, "a+")
-        file:write("")
-        file:close()
-        view:open_doc(doc_filename)
-        core.log("Created %s", doc_filename)
+        if filename:sub(#filename) == PATHSEP then
```

Comment:
Should consider `/` and `\` because it's annoying to do this on Windows.

Reply:
@takase1121 I thought that was the purpose of `PATHSEP` ? 
isn't it set to `/` or `\` depending on the operating system?
or are you saying it should just explicitly check for both that way you could use either character on both platforms?

Reply:
Windows supports both types of separators and since this is from user input, it might be better if the code can accept both.